### ISSUE TITLE
SwiftLintPlugin Hash mismatch for 0.63.0

### ISSUE THREAD

**Issue by Nef10** [realm/SwiftLint#6409]
### New Issue Checklist

- [x] I've Updated SwiftLint to the latest version.
- [x] I've searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues).

### Bug Description

Error:

```
error: checksum of downloaded artifact of binary target 'SwiftLintBinary' (396d5b609bf6ffc3e4d68b5279999c111dc3344a3a7e7b9b099243eaf46f1307) does not match checksum specified by the manifest (b51ca39ffe2331fe0337f9267d4b0dea2c182791a2fd0f3b961d7cbfb6d488d7))
```

Imported via Package.swift:

```swift
.package(url: "https://github.com/SimplyDanny/SwiftLintPlugins", exact: "0.63.0"),
```

It looks like in https://github.com/SimplyDanny/SwiftLintPlugins/commit/8c9743122ff1408d3a0acafd84707cbde5636218 only the checksum was updated, but not the download path. The checksum seems correct for the 0.63.0 release, but it still points to the rc2 version

**Comment by SimplyDanny** [comment]
Fixed. Thanks for the report!

**Comment by Nef10** [comment]
Hey @SimplyDanny did you just force push and override the tag?
Locally I needed to basically delete all SwiftPM caches before it got it to work without trying to download the old version.

However, it seems even GitHub Actions cached it and my runs are still failing with this error: https://github.com/Nef10/SwiftBeanCount/actions/runs/20700886449

**Comment by SimplyDanny** [comment]
Yes, I did. Sorry for that. It was the fastest way to get it working without triggering the whole build pipeline including Homebrew and Bazel updates only to fix the version in the plugins. I did not expect it to cause these massive problems.

**Comment by Nef10** [comment]
Ok, I guess I just wait for the next patch release for this to fix itself

**Comment by SimplyDanny** [comment]
> Ok, I guess I just wait for the next patch release for this to fix itself

[There](https://github.com/realm/SwiftLint/releases/tag/0.63.1) you go. Sorry again for the trouble.
